types/akamai-edgeworkers/index.d.ts

```diff
@@ -782,6 +785,38 @@ declare module "streams" {
         readonly writable: WritableStream<I>;
     }
 
+    const TransformStream: {
+        prototype: TransformStream;
+        new<I = any, O = any>(transformer?: Transformer<I, O>, writableStrategy?: QueuingStrategy<I>, readableStrategy?: QueuingStrategy<O>): TransformStream<I, O>;
+    };
+
+    interface Transformer<I = any, O = any> {
+        flush?: TransformerFlushCallback<O>;
+        readableType?: undefined;
```

Comment:
Are the `readableType` and `writableType` fields needed here? I'm not sure what they're for, although I see they're also present in the official TS dom typings. Maybe some kind of autogenerated annotation that leaked into the declarations?